File: Dev/Android/Codes/Prod/MyApplication/app/src/main/java/com/example/quentin/myapplication/structures/Area.java
package com.example.quentin.myapplication.structures;


public class Area {

	private String name;
	private Carte map;
	
	public Area(){
		name = "Default";
		map=new Carte();
		
	}
	// ***************** GETTERS *******************//
	
	public String getName() {
		return name;
	}
	
	// **************** SETTERS ****************//
	public void setName(String _name) {
		this.name = _name;
	}

	public Carte getMap() {
		return map;
	}

	public void setMap(Carte map) {
		this.map = map;
	}

	@Override
	public String toString() {
		return "Area [name=" + name + ", map=" + map + "]";
	}
	
}

File: Dev/PythonServer/structure/map/area.py
from map import Map
class Area:
    #Area object
    def __init__(self):
        self.name = ''
        self.map = Map()
    def parsing(self,value):
        if 'name' in value:   
            self.name = value['name']
        if 'map' in value:   
            self.map.parsing(value['map'])
    def __str__(self):
        return "Name : %s ; Map : %s" % (self.name,self.map)
